engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreSubmitActor.scala

```diff
@@ -71,11 +74,20 @@ final case class WorkflowStoreSubmitActor(store: WorkflowStore, serviceRegistryA
   
   override def receive = workflowStoreReceive.orElse(monitoringReceive)
 
-  private def storeWorkflowSources(sources: NonEmptyList[WorkflowSourceFilesCollection]): Future[NonEmptyList[WorkflowId]] = {
+  private def getWorkflowState(workflowStoreState: WorkflowStoreState): WorkflowState ={
```

Comment:
Maybe rename this something like `databaseStateToApiState`

Reply:
Renamed it to 'convertDatabaseStateToApiState'